src/Measurements.jl

```diff
@@ -101,6 +101,17 @@ measurement
 # Type representation
 Base.show(io::IO, measure::Measurement) =
```

Comment:
I think you can remove this method (or the method you defined can use the signature of this one)